silk/sql.py

```diff
@@ -77,7 +80,13 @@ def execute_sql(self, *args, **kwargs):
             return iter([])
         else:
             return
-    sql_query = q % tuple(force_str(param) for param in params)
+    str_params = []
+    for param in params:
+        try:
+            str_params.append(force_str(param))
+        except DjangoUnicodeDecodeError:
+            str_params.append(param)
```

Comment:
I'm curious about the implications of passing non-unicode-decodable data into the sql query, for example if the query will actually succeed when passed through (I doubt it would for python 2 given str/byte munging, though Django/Django-Silk doesn't support python 2 anymore), or if non-decodable data can be used for sql injection or other vulnerabilities (again, doubtful).

Reply:
The force_str was introduced in #617 .  We should keep that in.